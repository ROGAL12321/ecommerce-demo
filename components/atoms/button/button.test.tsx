import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'

import Button from './'

const ButtonComponent = <Button>test</Button>

describe('Button', () => {
  it('renders a button', () => {
    render(ButtonComponent)

    const button = screen.getByRole('button', {
      name: /test/i,
    })

    expect(button).toBeInTheDocument()
  })

  it('renders button unchanged', () => {
    const { container } = render(ButtonComponent)
    expect(container).toMatchSnapshot()
  })
})
