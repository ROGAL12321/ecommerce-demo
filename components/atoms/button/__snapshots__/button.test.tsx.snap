// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders button unchanged 1`] = `
<div>
  <button
    class="button"
  >
    test
  </button>
</div>
`;
